lib/features/login/data/local/login_repository_imp.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:fiura_ecosystem/features/login/domain/repositories/login_repository.dart';
+import 'package:google_sign_in/google_sign_in.dart';
+
+class LoginRepositoryImp extends LoginRepository {
+  //Collections
+  final String users = "users";
+
+  //Cloud firestore
+  final FirebaseFirestore db = FirebaseFirestore.instance;
+
+  //Firabase Auth
+  final GoogleSignIn googleSignIn = GoogleSignIn();
+  final FirebaseAuth auth = FirebaseAuth.instance;
```

Comment:
this should be provided on the constructor by the get it

Reply:
Fixed